src/Config.php

```diff
@@ -1658,4 +1644,62 @@ public function printConfigData($data)
     }//end printConfigData()
 
 
+    /**
+     * Assert that all supplied sniff codes have the correct number of parts
+     *
+     * @param string[] $sniffs   A list of sniffs supplied by the user, to be validated.
+     * @param string   $argument The name of the argument which is being validated.
+     *
+     * @return void
+     * @throws DeepExitException
+     */
+    private function validateSniffCodes($sniffs, $argument)
```

Comment:
```suggestion
    private function validateSniffCodes(array $sniffs, $argument)
```

Alternatively, it could be considered to make this method handle the CLI input string.
What I mean by that is:
* Leave the `substr()` call in the original logic;
* Pass the CLI args value (minus the prefix) as a string;
* Move the `explode()` to this method;
* And then if everything is valid, let this method return the validated array of sniff codes.